### PR TITLE
zbarcam: add page

### PR DESCRIPTION
**Pull request by Managor**
- [ x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ x] The page(s) have at most 8 examples.
- [ x] The page description(s) have links to documentation or a homepage.
- [ x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/zbarcam.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# zbarcam
+
+> Scans and decodes barcodes from a video device.
+> More information: <https://linux.die.net/man/1/zbarcam>.
```
Comment (pixelcmtd) [inline pages/linux/zbarcam.md:4]: It's really odd that their website doesn't have CLI docs. By default, we use `manned.org` for man pages.

```suggestion
> More information: <https://manned.org/zbarcam>.
```

File: pages/linux/zbarcam.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# zbarcam
+
+> Scans and decodes barcodes from a video device.
```
Comment (pixelcmtd) [inline pages/linux/zbarcam.md:3]: I think most people aren't gonna think of QR codes when they read that, so I'd prefer to clarify a bit

```suggestion
> Scans and decodes barcodes (and QR codes) from a video device.
```

File: pages/linux/zbarcam.md (hunk 1 above)
Comment (Managor) [reply pages/linux/zbarcam.md:4]: Alright, good to know. I haven't contributed in a while.

File: pages/linux/zbarcam.md (hunk 2 above)
Comment (Managor) [reply pages/linux/zbarcam.md:3]: I went to check that qr codes are called barcodes before committing, but I agree that most don't make the connection immediately.

**Review by pixelcmtd — APPROVED** [review]

**Review by kbdharun — COMMENTED** [review]
LGTM, Thanks for your contribution.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**